scripts/quarantine_zephyr.yaml

```diff
@@ -216,6 +216,28 @@
     - nrf5340dk/nrf5340/cpuapp/ns
   comment: "https://nordicsemi.atlassian.net/browse/NCSDK-31574"
 
+- scenarios:
+    - drivers.eeprom.emul.build
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuppr
+  comment: "https://nordicsemi.atlassian.net/browse/NCSDK-31589"
+
+- scenarios:
+    - llext.writable
+    - llext.writable_relocatable
+    - llext.writable_slid_linking
+    - llext.writable_relocatable_slid_linking
```

Comment:
there is already an entry forr llext above, can you pleas just add this platform there?